tests/unit/actions/test_default.py

```diff
@@ -88,7 +88,10 @@ def test_default_returns_callable_with_data(
     mocked_jira.create_or_update_issue_remote_links.return_value = sentinel
     mocked_bugzilla.get_bug.return_value = context_create_example.bug
     mocked_bugzilla.get_comments.return_value = []
-    callable_object = default.init(jira_project_key=context_create_example.jira.project)
+    callable_object = default.init(
+        jira_project_key=context_create_example.jira.project,
+        steps={"new": ["create_issue", "add_link_to_bugzilla"]},
```

Comment:
I'm curious, what was the motivation for this change?

Reply:
I was debugging a broken test and wanted to reduce the scope of executed steps. This is a left over, will revert